src/App.js

```diff
@@ -8,13 +8,54 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      loading: false,
+      movies: [],
       selectedMovie: 0,
+      error: "",
     };
   }
 
+  checkForError = (response) => {
+    if (response.ok) {
+      return response;
+    } else {
+      throw new Error(response.status);
+    }
+  };
+
+  componentDidMount() {
+    this.setState({ loading: true });
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then((response) => this.checkForError(response))
+      .then((response) => response.json())
+      .then((data) => {
+        this.setState({
+          loading: false,
+          movies: data.movies,
+        });
+      })
+      .catch((err) => this.setState({ error: err.message }));
```

Comment:
Really nice work checking for the error here and then setting the error message to state!  The only small refactor I might suggest is moving part of the fetch call into an `apiCalls.js` file along with the `checkForError` function and then importing & invoking them here.  Just a small suggestion!